library/Icingadb/Widget/Detail/ObjectDetail.php

```diff
@@ -248,7 +256,7 @@ protected function createNotes()
         if (empty($content)) {
             return null;
         }
-
+        
```

Comment:
Does this only look like as if there are spaces or are there actually spaces? If so, please remove them, this should be an *empty* line.